main.go

```diff
@@ -77,12 +76,31 @@ type encryptCloudImageBase struct {
 
 	workingDir string
 	devPath    string
+	devPathFormat string
 
 	partitions    gpt.Partitions
 	rootPartition *gpt.PartitionEntry
 	esp           *gpt.PartitionEntry
 }
 
+func (b *encryptCloudImageBase) initDevPathFormat(devPath string) error {
```

Comment:
should this be called something like `setDevPath` instead?

Reply:
Removed the initialization and replaced it with a function that will determine the format when it's needed.